### PR TITLE
open actionView to appropriate location when clicking on product text fields

### LINKED ISSUE #2999: Product Settings panel should open when clicking on the product title
### Expected behavior
Action view panel should open in context when clicking on the product title.

### Actual Behavior
Action view does not open when clicking product title.

### Steps to Reproduce the Behavior
- Go to PDP in edit mode.
- Click product title

### PR DESCRIPTION
**Pull request by kieckhafer**
Fixes #2999 

Opens PDP to Product Settings panel when loading the PDP for the first time, or clicking on a field taht is in the Product Settings section.

- As an admin, double click on a product from the product grid.
- See that the Product Settings panel is the open panel in the action view when landing on the PDP

-----

- As an admin, click the social settings edit button, so that the social section is open in the action view.
- Click the title / subtitle field on the PDP
- See that the correct section of the action view opens

### CODE REVIEW

**Comment by zenweasel** [comment]
@kieckhafer Did you mean to do this to `master`?

**Comment by kieckhafer** [comment]
@zenweasel did not, good catch, thanks

File: imports/plugins/core/dashboard/client/components/actionView.js [hunk 1]
```diff
@@ -465,6 +465,7 @@ class ActionView extends Component {
   }
 
   render() {
+    console.log("actionView", this.props);
```
Comment (spencern) [inline imports/plugins/core/dashboard/client/components/actionView.js]: Probably want to remove this?

File: imports/plugins/included/product-detail-simple/client/components/productField.js [hunk 2]
```diff
@@ -43,6 +45,14 @@ class ProductField extends Component {
     }
   }
 
+  handleFocus = (event) => {
+    // Open actionView, if it's not open already
+console.log("this input", this._input);
```
Comment (spencern) [inline imports/plugins/included/product-detail-simple/client/components/productField.js]: going to remove these for you as well.

**Comment by spencern** [comment]
@kieckhafer 
Here are the two problems that I'm still seeing. This first one is a little inconsistent, but by double-clicking the product in the product grid, I can reproduce it most of the time. I'm not sure if it's a double-click failure or if something is taking over once I land on the PDP. What happens is when I land on the PDP, the correct product panel is open in the action view, but then it immediately closes.


This second one is what I described above. When clicking the title while a variant edit panel is open, it does not open the product edit action view.

**Comment by kieckhafer** [comment]
@spencern issue 2 is fixed. Issue 1 still trying to figure out what triggers that closing.

**Comment by mikemurray** [comment]
@spencern The expanding issue should be resolved now.

**Comment by spencern** [comment]
Works as expected

**Pull request merged**